fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -430,10 +445,53 @@ case $DistortionCorrection in
                 --smoothingfwhm="2" \
                 --inputdir="$WD"
         fi
-            
+
 
         ;;
 
+    ${NONE_METHOD_OPT})
+
+            # NOTE: To work with later code a number of fake files are created 
+            #       with names that do not reflect the true nature of the file
+            #       this is to be considered an initial fix for datasets that 
+            #       do not include data that enable distortion correction and
+            #       will be replaced with a cleaner code. 
+
+
+            log_Msg "---> No distortion correction"
+
+            ScoutExtension=""
+
+            log_Msg "---> Copy Scout image"
+            ${FSLDIR}/bin/imcp ${ScoutInputName} ${WD}/${ScoutInputFile}${ScoutExtension}
+            
+            log_Msg "---> FAKE Jacobian Volume"
+            # Create fake Jacobian Volume for Regular Fieldmaps (all ones)
+            ${FSLDIR}/bin/fslmaths ${T1wImage} -mul 0 -add 1 -bin ${WD}/Jacobian.nii.gz
+            
+            log_Msg "---> register scout image to T1w"
+            # register scout image to T1w
+            # this is just an initial registration, refined later in this script, but it is actually pretty good
+            if [ $Preregister = "epi_reg" ] ; then
+                log_Msg "... running epi_reg (dof ${dof})"
+                ${HCPPIPEDIR_Global}/epi_reg_dof --dof=${dof} --epi=${WD}/${ScoutInputFile}${ScoutExtension} --t1=${T1wImage} --t1brain=${WD}/${T1wBrainImageFile} --out=${WD}/${ScoutInputFile}${ScoutExtension}2T1w_init
+            elif [ $Preregister = "flirt" ] ; then
+                log_Msg "... running flirt"
+                ${FSLDIR}/bin/flirt -in ${WD}/${ScoutInputFile}${ScoutExtension} -ref ${WD}/${T1wBrainImageFile} -out ${WD}/${ScoutInputFile}${ScoutExtension}2T1w_init -omat ${WD}/${ScoutInputFile}${ScoutExtension}2T1w_init.mat -dof 6
+            fi
+
+            #copy the initial registration into the final affine's filename, as it is pretty good
+            #we need something to get between the spaces to compute an initial bias field
+            cp "${WD}/${ScoutInputFile}${ScoutExtension}2T1w_init.mat" "${WD}/fMRI2str.mat"
+
+            # generate Scout2T1 warpfield and spline interpolated images + apply bias field correction
+            log_Msg "generate combined warpfields and spline interpolated images and apply bias field correction"
+            ${FSLDIR}/bin/convertwarp --relout --rel -r ${T1wImage} --premat=${WD}/${ScoutInputFile}${ScoutExtension}2T1w_init.mat -o ${WD}/${ScoutInputFile}${ScoutExtension}2T1w_init_warp
```

Comment:
add a comment preceding this line:
```
# In the NONE condition, we have no distortion Warpfield.  Convert the Scout2T1 registration (affine)
# to its warp field equivalent, since we need "${WD}/${ScoutInputFile}${ScoutExtension}2T1w_init_warp" later
```

Reply:
Added.